edgelet/iotedge/src/check/mod.rs

```diff
@@ -134,7 +134,7 @@ impl Check {
                     all_checks.extend(
                         aziot_checks
                             .into_iter()
-                            .map(|(section_name, checks)| (section_name + " (aziot)", checks)),
+                            .map(|(section_name, checks)| (section_name + " (aziotctl)", checks)),
```

Comment:
Well it's the section name rather than the binary name, so `aziot-identity-service` to indicate the package, perhaps.

Reply:
Fixed

Reply:
https://github.com/Azure/iotedge/pull/4596/files#diff-5b05b8525be1ca907a0bc313fd71312d95a964b5f60d1903aa0fc2992a247027R147 etc

Reply:
Changed all instances of aziotctl to aziot-identity-service.